modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -75,13 +77,15 @@ struct GAPI_Streaming: public ::testing::TestWithParam<KernelPackage> {
         {
         case KernelPackage::OCV:
             return cv::gapi::combine(core::cpu::kernels(),
-                                     imgproc::cpu::kernels());
+                                     imgproc::cpu::kernels(),
+                                     streaming::kernels());
```

Comment:
No.

One shouldn't pass `streaming::kernels()`. I'd expect these kernels to be available _always_ (= included into the compiler's default kernel package, regardless of `use_only{}`).

Reply:
Removed